spec/motion-spec/mac_bacon_spec.rb

```diff
@@ -137,24 +136,24 @@ def triggerChange
           end
         end
 
-        behaves_like "waiting in before/after filters"
+        behaves_like 'waiting in before/after filters'
       end
 
-      describe "and without explicit time" do
+      describe 'and without explicit time' do
         before do
           @started_at ||= Time.now
-          performSelector('resume', withObject:nil, afterDelay:0.5)
+          performSelector('resume', withObject: nil, afterDelay: 0.5)
           wait do
-            performSelector('resume', withObject:nil, afterDelay:0.5)
+            performSelector('resume', withObject: nil, afterDelay: 0.5)
             wait do
             end
           end
         end
 
         after do
-          performSelector('resume', withObject:nil, afterDelay:0.5)
+          performSelector('resume', withObject: nil, afterDelay: 0.5)
           wait do
-            performSelector('resume', withObject:nil, afterDelay:0.5)
+            performSelector('resume', withObject: nil, afterDelay: 0.5)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.